src/nvim/testdir/test_cursor_func.vim

```diff
@@ -66,7 +66,6 @@ func Test_curswant_with_cursorline()
 endfunc
 
 func Test_screenpos()
```

Comment:
This test passes for `TEST_FILE=test_cursor_func make oldtest` but fails in executed from `test_alot.vim`.